libraries/AC_PrecLand/AC_PrecLand.h

```diff
@@ -6,6 +6,10 @@
 #include "PosVelEKF.h"
 #include <AP_HAL/utility/RingBuffer.h>
 
+#define EKF_INIT_TIME_MS 2000
```

Comment:
Better make those static const inside the class. That will limit the usage scope to the class. Here with those kind of generic name define there is a risk of collision with other lib.